components/tools/OmeroPy/test/integration/gatewaytest/test_config_service.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+   Copyright (C) 2016 University of Dundee & Open Microscopy Environment.
+                      All Rights Reserved.
+   Copyright 2013 Glencoe Software, Inc. All rights reserved.
+   Use is subject to license terms supplied in LICENSE.txt
+
+   pytest fixtures used as defined in conftest.py:
+   - gatewaywrapper
+
+"""
+
+
+def testInterpolateSetting(gatewaywrapper):
+    """
+    Tests conn.getInterpolateSetting(), setting the value as Admin and
+    testing the output as a regular user.
+    """
+    gatewaywrapper.loginAsAdmin()
+    configService = gatewaywrapper.gateway.getConfigService()
+
+    try:
+        configService.setConfigValue(
+            "omero.client.viewer.interpolate_pixels", "false")
+
+        gatewaywrapper.loginAsAuthor()
+        assert not gatewaywrapper.gateway.getInterpolateSetting()
+
+    finally:
+        # try/finally to make sure that we
+        # set back to 'true' (default value) to clean up
+        gatewaywrapper.loginAsAdmin()
+        configService = gatewaywrapper.gateway.getConfigService()
+        configService.setConfigValue(
+            "omero.client.viewer.interpolate_pixels", "true")
```

Comment:
you should use ``getConfigService().getConfigDefaults()`` to reset to what is default rather then hard code it. IF default change this test will affect others.